django/cantusdb_project/templates/base.html

```diff
@@ -325,7 +325,7 @@ <h5>Contributor Navigation</h5>
                                 {% endif %}
                                 {% if request.user|has_group:"project manager" %}
                                     <li class="list-group-item p-0">
-                                        <a href="">Add News</a>
+                                        <a href="/admin/articles/article/add/">Add News</a>
```

Comment:
Is it possible to avoid hard-coding this url? Does `{% url ... %}` work?

Reply:
I'm not certain, but I think that to link to the Admin panel, we have to hard-code it. I think if we had an article-create view, we could do it with `{% url ... %}`, but we don't so we can't.

Reply:
this is the same format we use a couple of lines later, for "Add Feast", "Add Indexer", etc.

Reply:
Can you look into this? https://docs.djangoproject.com/en/4.0/ref/contrib/admin/#reversing-admin-urls
If possible, we should replace all hard-coded urls to Django admin interface with url tags.